cli/src/common/utils.ts

```diff
@@ -0,0 +1,8 @@
+export async function asyncForEach<T>(
```

Comment:
Worth adding some JSDoc here, as one could expect that it runs in parallel.

Note before you address this, I have another suggestion to delete this function altogether below.